client/packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -82,9 +83,9 @@ const getNextStatusOption = (
 ): SplitButtonOption<InvoiceNodeStatus> | null => {
   if (!status) return options[0] ?? null;
 
-  const nextStatus = getNextOutboundStatus(status);
-  const nextStatusOption = options.find(o => o.value === nextStatus);
-  return nextStatusOption || null;
+  const currentIndex = options.findIndex(o => o.value === status);
+  const nextOption = options[currentIndex + 1];
+  return nextOption || null;
```

Comment:
Getting the next one can be done using the current options list, no need to call another function

Reply:
Agreed for now, I think conceptually the function might be able to handle a forking workflow. e.g. if say it's marked as incomplete, maybe the next step changes. I don't think we actually have that though so this is much simpler.